### PR TITLE
Open tickets-sold summary on long pressing Event list item

### LINKED ISSUE #1059: Long pressing Event List Items should show tickets sold summary
**Is it a ?**

- [ ] Bug Report

- [x] Feature Request
    
- [ ] Chore
    
### Feature Request

**Feature Description**

In the EventBrite app, Long Pressing Event List items shows ticket sales summary. Similar feature should be present in the Organizer App.
The 'tickets sold' layout in Dashboard is also vertical in EventsBrite app, but is horizontal in Open Event App. 
Using vertical layout at both places would allow reusing a common again.

videotogif_2018 06 11_22 08 16

**Would you like to work on the issue?**

Yes

### PR DESCRIPTION
**Pull request by mishuvs**
Fixes #1059 

Changes:
1. Long pressing Event List items shows tickets sold summary
2. Similar layout has replaced the older horizontal view for Dashboard.

Screenshots for the change:

### CODE REVIEW

**Comment by mishuvs** [comment]
@Masquerade0097 @sridharjajoo
The value for `event.analytics.freeTickets.get()` is coming out to be `0` in my fragment, but it comes out to be `100` when used in dashboard. The `event.name` comes out correct. Any idea why `analytics` isn't working?

**Comment by mishuvs** [comment]
@sridharjajoo @Masquerade0097 @iamareebjamal Please review

**Comment by mishuvs** [comment]
This is to make it consistent with the EventBrite App, which has it vertical in both. 
Besides that, in the last meeting Mario had pointed out that EventBrite has already worked on, and spent money on designs, and we don't need to redo that.
And what happens if people say that: "EventBrite has it better, it doesn't look good" ? Then we'd have to change it again.
So I am trying to stick with Mario's advice and not tinker with the designs, so as to put total efforts in developing.

**Comment by Masquerade0097** [comment]
@mishuvs It's fine, if Mario says so. I was just giving my opinion. 
Also, you haven't mentioned that changing Dashboard layout was also part of this issue neither in Issue tracker nor in this PR description. Please write the complete description of the issue in the issue tracker so that everyone knows what changes are expected for the issue.

File: app/src/main/java/org/fossasia/openevent/app/core/event/list/EventsListAdapter.java [hunk 1]
```diff
@@ -77,8 +80,9 @@ public EventRecyclerViewHolder onCreateViewHolder(ViewGroup parent, int viewType
         EventLayoutBinding binding = EventLayoutBinding.inflate(layoutInflater, parent, false);
         EventRecyclerViewHolder eventRecyclerViewHolder = new EventRecyclerViewHolder(binding);
 
-        eventRecyclerViewHolder.onItemLongClick(eventsPresenter::toolbarEditMode);
```
Comment (sridharjajoo) [inline app/src/main/java/org/fossasia/openevent/app/core/event/list/EventsListAdapter.java]: As you are removing the option to edit event on long press, make sure that the code related to it such as the toolbarEditMode( ) in the EventsPresenter also gets removed.

**Comment by mishuvs** [comment]
@sridharjajoo @Masquerade0097 @iamareebjamal Please review

**Comment by mishuvs** [comment]
Sorry for pushing again.. only added "Ticekts Sold" TextView to `fragment_sales_summary.xml`. Screenshot updated.

**Pull request merged**